packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -256,6 +266,17 @@ export default compose(
 			clipboardBlock &&
 			canInsertBlockType( clipboardBlock.name, rootClientId );
 
+		// STOPSHIP (blixt): Under what conditions is the transform option
+		// enabled? I'm guessing we'll start with "if the block-type is
+		// Paragraph" for now? Is that something like 
+		// blockType === 'core/paragraph'; 
+		// Poking around in getBlockType, I'm not sure what strings get 
+		// returned from this, and I'm hesitant to hard-code a string here.
```

Comment:
Yup `core/paragraph` is the block type for the paragraph 👍  

[block-library](https://github.com/WordPress/gutenberg/tree/master/packages/block-library) contains more information on individual blocks if you need to compare where values are being mapped from.